test_httpbin_ip.py

```diff
@@ -153,3 +161,37 @@ def test_redirect_to_absolute(config):
     assert r.history[0].url == config['base_url'] + 'absolute-redirect/1'
     assert r.json()['args'] == {}
     assert r.json()['headers']['User-Agent'] == 'python-requests/' + str(requests.__version__)
+
+
+@pytest.mark.parametrize('img_type, img_name, content_type', [
+    ('jpeg', 'wolf.jpg', 'jpeg'),
+    ('png', 'pig.png', 'png'),
+    ('webp', 'webp', 'webp'),
+    ('svg', '1.svg', 'svg+xml'),
+])
+def test_img(config, base_url, img_type, img_name, content_type):
+    r = get(config['base_url'] + 'image', headers={'Accept': 'image/' + content_type})
+
+    assert r.status_code == 200
+    assert r.elapsed.total_seconds() < 10
+    assert r.reason == 'OK'
+    assert r.headers.get('Content-Type') == 'image/' + content_type
+
+    filename = img_name
+    path_to_current_file = os.path.realpath(__file__)
+    current_directory = os.path.split(path_to_current_file)[0]
+    data_file_path = os.path.join(current_directory, "images", filename)
+    file_binar_data = open(data_file_path, 'rb').read()
+
+    assert r.content == file_binar_data
+
+    with open('open_temp_file', 'wb') as file:
+        file.write(r.content)
```

Comment:
still need to rename `open_temp_file` to `temp_file`.
open_temp_file is a name for action like a method, but here it should be just a noun to name the temporary file (e.g. temp_file, tempfile, dummy_file, etc.)
Additionally don't use built-in keywords as variable names. `file` is a reserved word in Python to denote a type of a file object. I usually just use `f`, you can use `myfile` or `tempfile`.